docs/root/development/performance/cpu_battery_impact.rst

```diff
@@ -0,0 +1,150 @@
+.. _dev_performance_cpu_battery:
+
+Analysis of CPU/battery impact
+==============================
+
+Modified versions of the "hello world" example apps were used to run these experiments:
+
+- `Android control app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/kotlin/control>`_
+- `Android Envoy app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/kotlin/hello_world>`_
+- `iOS control app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/objective-c/control/control>`_
+- `iOS Envoy app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/objective-c/xcode_variant/EnvoyObjc/EnvoyObjc>`_
+
+These apps made network requests every ``200ms`` with all request/response caching disabled.
+
+Results
+~~~~~~~
+
+iOS
+---
+
+Envoy:
+
+- Avg CPU: >= 100%
+- Avg memory: 12MB
+- Battery: 12/20 Xcode Instruments score
+
+Control:
+
+- Avg CPU: 12%
+- Avg memory: 6MB
+- Battery: 1/20 Xcode Instruments score
+
+**Based on these results, memory usage is similar. However, CPU (and consequently battery) usage is very high.**
+The root cause has been identified and is being tracked in `issue 215 <https://github.com/lyft/envoy-mobile/issues/215>`_.
+
+Android
+-------
+
+Envoy:
+
+- Avg CPU: 33.16075949%
+- Avg memory: 2.765822785%
+- Battery: 0.17%/min
+
+Control:
+
+- Avg CPU: 28.81012658%
+- Avg memory: 2.169620253%
+- Battery: 0.18%/min
+
+**Based on these results, control and Envoy are relatively similar.**
+
+Experimentation method
+~~~~~~~~~~~~~~~~~~~~~~
+
+iOS
+---
+
+The original investigation was completed as part of `this issue <https://github.com/lyft/envoy-mobile/issues/113>`_.
+
+For analysis, the `Energy Diagnostics tool from Xcode Instruments <https://developer.apple.com/library/archive/documentation/Performance/Conceptual/EnergyGuide-iOS/MonitorEnergyWithInstruments.html>`_
+was used.
+
+The analysis utilized two apps:
+
+- **Control** - An app that makes a request every ``200ms`` to an endpoint without Envoy compiled in the app
```

Comment:
Should we pull this up out of the platform specific sections since this applies to both?

Reply:
Sure, done